config/routes.rb

```diff
@@ -2,9 +2,11 @@
 
   get "/" => "home#index"
```

Comment:
I prefer the new notation (with to:) http://guides.rubyonrails.org/routing.html#connecting-urls-to-code